test/check/check.go

```diff
@@ -808,6 +816,21 @@ func checkForNetworkAttachmentDefinition(name string) error {
 	return nil
 }
 
+func checkForConfigMap(name string) error {
```

Comment:
Since this template repeats, potential follow up (didn't validate)
wdty ?

```
func checkForObject(name, kind, namespace string, obj client.Object) error {
	err := testenv.Client.Get(context.Background(), types.NamespacedName{Name: name, Namespace: namespace}, obj)
	if err != nil {
		return err
	}

	err = checkRelationshipLabels(obj.GetLabels(), kind, name)
	if err != nil {
		return err
	}

	return nil
}
```